irohad/multi_sig_transactions/transport/impl/mst_transport_grpc.cpp

```diff
@@ -108,9 +109,8 @@ void MstTransportGrpc::sendState(const shared_model::interface::Peer &to,
   peer->set_address(to.address());
   for (auto &batch : providing_state.getBatches()) {
     for (auto &tx : batch->transactions()) {
-      auto addtxs = protoState.add_transactions();
       // TODO (@l4l) 04/03/18 simplify with IR-1040
-      new (addtxs) protocol::Transaction(
+      *protoState.add_transactions() = protocol::Transaction(
```

Comment:
No need for additional `protocol::Transaction()`, as it will make a copy anyway.